src/constants/src/default-settings.ts

```diff
@@ -315,47 +315,57 @@ export const DEFAULT_MAP_STYLES: {
   {
     id: 'dark',
     label: 'Dark',
-    url: 'mapbox://styles/uberdata/cjoqbbf6l9k302sl96tyvka09',
+    url: 'https://api.maptiler.com/maps/b989b458-e075-497d-9a86-3035fd08bd7c/style.json?key=17oUJVO01Blh1QOfWftj',
     icon: `${BASEMAP_ICON_PREFIX}/UBER_DARK_V2.png`,
     layerGroups: DEFAULT_LAYER_GROUPS,
     colorMode: BASE_MAP_COLOR_MODES.DARK,
     complimentaryStyleId: 'light'
   },
   {
-    id: 'light',
-    label: 'Light',
-    url: 'mapbox://styles/uberdata/cjoqb9j339k1f2sl9t5ic5bn4',
-    icon: `${BASEMAP_ICON_PREFIX}/UBER_LIGHT_V2.png`,
-    layerGroups: DEFAULT_LAYER_GROUPS,
-    colorMode: BASE_MAP_COLOR_MODES.LIGHT,
-    complimentaryStyleId: 'dark'
-  },
-  {
-    id: 'muted',
-    label: 'Muted Light',
-    url: 'mapbox://styles/uberdata/cjfyl03kp1tul2smf5v2tbdd4',
-    icon: `${BASEMAP_ICON_PREFIX}/UBER_MUTED_LIGHT.png`,
-    layerGroups: DEFAULT_LAYER_GROUPS,
-    colorMode: BASE_MAP_COLOR_MODES.LIGHT,
-    complimentaryStyleId: 'muted_night'
-  },
-  {
-    id: 'muted_night',
-    label: 'Muted Night',
-    url: 'mapbox://styles/uberdata/cjfxhlikmaj1b2soyzevnywgs',
-    icon: `${BASEMAP_ICON_PREFIX}/UBER_MUTED_NIGHT.png`,
+    id: 'dark-matter',
+    label: 'Dark Matter',
+    url: 'https://basemaps.cartocdn.com/gl/dark-matter-gl-style/style.json',
+    icon: `${BASEMAP_ICON_PREFIX}/UBER_DARK_V2.png`,
     layerGroups: DEFAULT_LAYER_GROUPS,
     colorMode: BASE_MAP_COLOR_MODES.DARK,
-    complimentaryStyleId: 'muted'
+    complimentaryStyleId: 'light'
   },
-  {
-    id: 'satellite',
-    label: 'Satellite',
-    url: `mapbox://styles/mapbox/satellite-v9`,
-    icon: `${BASEMAP_ICON_PREFIX}/UBER_SATELLITE.png`,
-    layerGroups: [],
-    colorMode: BASE_MAP_COLOR_MODES.NONE
-  }
+
+  // {
```

Comment:
@macrigiuseppe I assume we do need counterparts to all of these in order to load saved maps?